pinot-hadoop/src/main/java/com/linkedin/pinot/hadoop/job/SegmentCreationJob.java

```diff
@@ -211,29 +212,48 @@ public void run() throws Exception {
   }
 
   protected void setAdditionalJobProperties(Job job) throws Exception {
+    // Update push locations
```

Comment:
Can we change this comment? Update seems like it's a different push location, maybe add?